### PR TITLE
Flow 7 compatible

### PR DESCRIPTION
**Pull request by sorenmalling**
Changes made

- Adjusted migration namespace
- Removed routing setting `before Neos.Neos` since changes in positionalarraysorter that throws a exception if <key> does not exists
- Adjusted composer.json requirement due to new required doctrine/migrations

### CODE REVIEW

File: Configuration/Settings.Neos.yaml [hunk 1]
```diff
@@ -2,6 +2,4 @@ Neos:
   Flow:
     mvc:
       routes:
-        PunktDe.OAuth2.Server:
```
Comment (daniellienert) [inline Configuration/Settings.Neos.yaml:5]: Correct (as the package does not require Neos), but breaking as AFAIK it won't work like this when Neos is installed. We should mention that in the Readme.

**Review by daniellienert — COMMENTED** [review]
Big thanks for taking care @sorenmalling. I just merged the other PRs would you rebase this one?

File: Configuration/Settings.Neos.yaml (hunk 1 above)
Comment (sorenmalling) [reply Configuration/Settings.Neos.yaml:5]: @daniellienert Or add "position: start"?

Comment (daniellienert) [reply Configuration/Settings.Neos.yaml:5]: Even better!!

**Comment by sorenmalling** [comment]
@daniellienert My rebase skills might not be up to date - I'm a bid unsure if I got master wrongly merged/rebased here 😬

**Comment by daniellienert** [comment]
@sorenmalling, I did the rebase for you and changed the route positioning

**Pull request merged**